### PR TITLE
Components

### PR DESCRIPTION
**Pull request by davidwobido**
**25.11.2021**
• Add Festival card large
• Add Festival card medium
• Add Festival card small

image
image
image

**24.11.2021**
• Add buttons (default, wide, small)
• Add genre tag
• Add search bar

image
image
image
image
image

### CODE REVIEW

File: src/components/Buttons/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import styles from "./Button.module.css";
+import "../../globals.css";
+
+type ButtonProps = {
+  size: string;
```
Comment (lmachens) [inline src/components/Buttons/Button.tsx:5]: You could make sure that you don't add a not-existing size

```suggestion
  size: 'normal' | 'wide' | 'small';
```

File: src/components/SearchBar/SearchBar.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+type SearchBarProps = {
+  onSearch: (value: string) => void;
+};
+
+function SearchBar({ onSearch }: SearchBarProps): JSX.Element {
+  const [value, setValue] = useState("");
+
+  console.log(value);
+  console.log(onSearch);
```
Comment (lmachens) [inline src/components/SearchBar/SearchBar.tsx:12]:
```suggestion
```

File: src/components/SearchBar/SearchBar.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  return (
+    <form className={styles.wrapper}>
+      <input
+        value={value}
+        onChange={(event) => setValue(event.target.value)}
+        placeholder="Search"
+        className={styles.input}
+      />
+      <button type="submit" className={styles.button} value="">
```
Comment (lmachens) [inline src/components/SearchBar/SearchBar.tsx:22]:
```suggestion
      <button type="submit" className={styles.button}>
```

**Review by lmachens — COMMENTED** [review]
Good job!

File: src/components/SearchBar/SearchBar.tsx (hunk 2 above)
Comment (davidwobido) [reply src/components/SearchBar/SearchBar.tsx:12]: Would result in an error and I’ll need them later.

File: src/components/Buttons/Button.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import Button from "./Button";
+
+export default {
+  component: Button,
+  title: "Components/Button",
+};
+
+export const Normal = () => <Button size="normal" text="Filter" />;
+export const Wide = () => <Button size="wide" text="hallo" />;
```
Comment (riittagerhards) [inline src/components/Buttons/Button.stories.tsx:9]: This is the only button starting with a lower case letter. Here it doesn't matter, but might be important for later.

**Review by riittagerhards — COMMENTED** [review]
LGTM 👍

**Pull request merged**